app/views/home/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container-fluid" ng-app="receta">
+    <div class="panel panel-success">
+        <div class="panel-heading">
+            <h1 ng-if="name">Hello, {{name}}</h1>
```

Comment:
We can leave it empty as long as it doesn't contain this irrelevant stuff. We'll need the file anyway so as long as it's empty it's fine.